app/src/renderer/components/network/PageNetwork.vue

```diff
@@ -1,7 +1,12 @@
 <template>
-  <tm-page data-title="Network">
-    <tm-data-error v-if="!connected" />
-    <template v-else>
+  <tm-page
+    :loading="blocks.loading"
+    :loaded="blocks.loaded"
+    :error="blocks.error"
+    :dataset="lastHeader"
```

Comment:
page network is a bit strange because it doesn't need the "items" (dataset) to load. it just needs a `connection` to render the top part of the page. the `blocks` are the second "dataset". `lastHeader` and `connection` don't have loading, loaded, or error states...

reveals room for a refactor - but this might be sufficient for now.

Reply:
I think we have in the store the list of blocks, but I like more @faboweb idea of booleanize

Reply:
me too